TOOODO/Data/Sources/Dto/Local/CategoryDto.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CategoryDto.swift
+//  Data
+//
+//  Created by 노원진 on 11/5/24.
+//
+
+import RealmSwift
+import Domain
+
+public class CategoryDto: Object {
+    @Persisted(primaryKey: true) var id: ObjectId
+    @Persisted var category: String
+    @Persisted var categoryDes: String
+    @Persisted var color: Int
+    @Persisted var todos: List<TodoDto>
+}
+
+extension CategoryDto {
+    func toEntity() -> CategoryEntity {
```

Comment:
```suggestion
    func toDomain() -> CategoryEntity {
```
도메인을 많이 쓰지만 큰 문제는 아니에요